libraries/core_libs/storage/src/storage.cpp

```diff
@@ -48,9 +49,12 @@ DbStorage::DbStorage(fs::path const& path, uint32_t db_snapshot_each_n_pbft_bloc
   // https://github.com/facebook/rocksdb/issues/3216#issuecomment-817358217
   // aleth default 256 (state_db is using another 128)
   options.max_open_files = (max_open_files) ? max_open_files : 256;
-  std::vector<rocksdb::ColumnFamilyDescriptor> descriptors;
+
+  std::vector<rocksdb::ColumnFamilyDescriptor> descriptors(Columns::all.size());
   std::transform(Columns::all.begin(), Columns::all.end(), std::back_inserter(descriptors), [](const Column& col) {
-    return rocksdb::ColumnFamilyDescriptor(col.name(), rocksdb::ColumnFamilyOptions());
+    auto options = rocksdb::ColumnFamilyOptions();
+    options.comparator = getUintComparator();
```

Comment:
I think we dont need it for all columns. Only for `sortition_params_change` column

Reply:
this was just to test, your test